core/src/main/java/com/flipkart/varadhi/services/OrgService.java

```diff
@@ -0,0 +1,38 @@
+package com.flipkart.varadhi.services;
+
+import com.flipkart.varadhi.db.MetaStore;
+import com.flipkart.varadhi.entities.*;
+import com.flipkart.varadhi.exceptions.DuplicateResourceException;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+public class OrgService implements VaradhiEntityService<OrgEntity> {
+
+    private final MetaStore metaStore;
+
+    private static final VaradhiEntityType VARADHI_ENTITY_TYPE = VaradhiEntityType.ORG;
+
+    public OrgService(MetaStore metaStore) {
+        this.metaStore = metaStore;
+    }
+
+    @Override
+    public OrgEntity create(OrgEntity orgEntity) {
+        log.info("Creating Org entity {}", orgEntity.getName());
+
+        if (exists(orgEntity.getName())) {
+            log.error("Specified Org({}) already exists.", orgEntity.getName());
```

Comment:
we can remove log.error(). Following the convention where only log exception if they are being handled. Unhandled exception will be logged by default failure handler at top of the chain.